src/main/resources/templates/probes/listing.html

```diff
@@ -2,25 +2,31 @@
 <html lang="en" data-layout-decorate="~{layouts/main}">
 <head>
     <title>Probes</title>
+    <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.19/css/jquery.dataTables.min.css">
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/datatables/1.10.19/js/jquery.dataTables.min.js"></script>
+    <script src="" data-th-src="@{/js/index.js}" type="text/javascript"></script>
```

Comment:
No, this is loaded in the layout `main`

```suggestion
```

I believe that you needed to do that because, when the code in the `index.js` file is executed, neither `jquery` nor `datatables` librairies were loaded so the code is not executing correctly.

Reply:
Yes, that was the reason.